app/views/reports/dashboards/overviews/details.haml

```diff
@@ -0,0 +1,25 @@
+- if @sub_key.present?
+  - title = " #{@sub_key} #{@section.to_s.humanize} (#{@key})".titlecase
+-else
+  - title = " #{@key} #{@section.to_s.humanize}".titlecase
+- content_for :title, title
+= content_for :modal_title, title
+
+%table.table
+  %thead
+    - details_columns.each do |name|
+      %th= name.to_s.humanize
+  %tbody
+    - @clients.each do |client|
+      %tr
+        - if client.accessible_by_user?(current_user)
```

Comment:
This is going to be a relatively expensive N+1.  Can we figure out how to select it as a column or in a batch and pre-fetch `Client.where(id: client_ids).visible_by(user).pluck(:id)` and just do a lookup?

Reply:
Fixed.